packages/widget-core/src/__tests__/assign-configuration.spec.ts

```diff
@@ -1,9 +1,29 @@
 import { assignConfiguration } from '../assign-configuration'
-import type { ExtendedWindow } from '../types'
+import type { ExtendedWindow, Token } from '../types'
 
 declare const window: ExtendedWindow
 
+const exampleToken: Token = {
+	accessToken:
+		'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c',
+	entityId: 123456,
```

Comment:
It should be adjusted after changing the type of Token, for example to random uuid:

```suggestion
	entityId: 'ec3469fc-5f12-4d05-a7a8-48baecd30208',
```